readme.md

```diff
@@ -1,5 +1,13 @@
 # Sijot website.
 
+## Info
+This website belongs to a local scouting group at St.George. The website provide information such as :-
```

Comment:
thanks @Tjoosten for your suggestions. I have made a new PR with the changes you told.